uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/models/calendar-model.js

```diff
@@ -28,9 +28,7 @@ define([
 
           return workPatternAPI.getCalendar(contactId, periodId, params)
             .then(function (data) {
-              return instance.init({
-                days: data.values
```

Comment:
is `days` keys used by any other dependent code? I remember it using in `LeaveRequestPopupCtrl`